.env

```diff
@@ -0,0 +1,5 @@
+DB_HOST=localhost
```

Comment:
I added the .env to the repo just to have an example of how the file can be used in a golang project to use environment variables.